regression/cases/file_to_blackhole/lading/lading.yaml

```diff
@@ -8,3 +8,7 @@ generator:
       bytes_per_second: "100Mb"
       maximum_bytes_per_file: "100Mb"
       maximum_prebuild_cache_size_bytes: "400Mb"
+
+blackhole:
+  - tcp:
+      binding_addr: "127.0.0.1:15400"
```

Comment:
I guess I'm confused by this - what writing to this addr?

Reply:
This is exposing a black hole in lading so that Vector can sink into it. Without this there's no egress throughout to measure and as the file generator can't participate in back pressure with Vector the ingress throughout measure is worthless.

Reply:
That means we should actually sink to that address? Right now it's still running the `blackhole` sink, unless I'm confused